redaxo/src/addons/debug/lib/debug_clockwork.php

```diff
@@ -13,6 +13,8 @@ private static function init(): void
     {
         $clockwork = \Clockwork\Support\Vanilla\Clockwork::init([
             'storage_files_path' => self::getStoragePath(),
+            'storage_files_compress' => true,
+            'storage_expiration' => 60 * 24 * 2,
```

Comment:
bewusst 2 tage ausgewählt, da es im cleanup eine Chance von 1 zu 100 gibt, dass der das cleanup auch macht (in Clockwork so gelöst)

Reply:
Gerne im code als kommentar drauf hinweisen